chihuahua/assetlist.json

```diff
@@ -106,7 +106,107 @@
           "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/chihuahua/images/woof.png"
         }
       ]
-    }    
+    },
+    {
+      "description": "Tacos",
+      "denom_units": [
+        {
+          "denom": "factory/chihuahua13jawsn574rf3f0u5rhu7e8n6sayx5gkw3eddhp/utacos",
+          "exponent": 0
+        },
+        {
+          "denom": "TACOS",
+          "exponent": 6
+        }
+      ],
+      "base": "factory/chihuahua13jawsn574rf3f0u5rhu7e8n6sayx5gkw3eddhp/utacos",
+      "name": "TACOS",
+      "display": "TACOS",
+      "symbol": "TACOS",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/chihuahua/images/tacos.png"
+      },
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/chihuahua/images/tacos.png"
+        }
+      ]
+    },
+    {
+      "description": "Weed",
+      "denom_units": [
+        {
+          "denom": "factory/chihuahua13jawsn574rf3f0u5rhu7e8n6sayx5gkw3eddhp/uweed",
+          "exponent": 0
+        },
+        {
+          "denom": "WEED",
+          "exponent": 6
+        }
+      ],
+      "base": "factory/chihuahua13jawsn574rf3f0u5rhu7e8n6sayx5gkw3eddhp/uweed",
+      "name": "WEED",
+      "display": "WEED",
+      "symbol": "WEED",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/chihuahua/images/weed.png"
+      },
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/chihuahua/images/weed.png"
+        }
+      ]
+    },
+    {
+      "description": "Bulldog",
+      "denom_units": [
+        {
+          "denom": "factory/chihuahua13jawsn574rf3f0u5rhu7e8n6sayx5gkw3eddhp/ubdog",
+          "exponent": 0
+        },
+        {
+          "denom": "BDOG",
+          "exponent": 6
+        }
+      ],
+      "base": "factory/chihuahua13jawsn574rf3f0u5rhu7e8n6sayx5gkw3eddhp/ubdog",
+      "name": "BDOG",
+      "display": "BDOG",
+      "symbol": "BDOG",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/chihuahua/images/bdog.png"
+      },
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/chihuahua/images/bdog.png"
+        }
+      ]
+    },
+    {
+      "description": "Bulldog",
```

Comment:
Is this description perhaps incorrect? Copied from the Bulldog token and not updated for this one?

Reply:
True! Fixed here: https://github.com/cosmos/chain-registry/pull/3669/commits/be33c36f898a8d125f64478b9d580f435408b7ef
Thanks o/